documentation/sphinx/source/disk-snapshot-backup.rst

```diff
@@ -0,0 +1,90 @@
+
+.. _disk-snapshot-backups:
+
+#################################
+Disk Snapshot Backup and Restore
+#################################
+
+This document covers disk snapshot based backup and restoration of a FoundationDB database. This tool leverages disk level snapshots and gets a point-in-time consistent copy of the database. The disk snapshot backup can be used to provide additional level of protection in case of hardware or software failures, test and development purposes or for compliance reasons.
+
+.. _disk-snapshot-backup-introduction:
+
+Introduction
+============
+
+FoundationDB's disk snapshot backup tool makes a consistent, point-in-time backup of FoundationDB database without downtime by taking crash consitent snapshot of all the disk stores that has persistent data.
+
+Crash consitent snapshot feature at file-system or disk level is a pre-requsiste for using this feature.
```

Comment:
I prefer the following rephrase:
The prerequisite of this feature is that the filesystem FoundationDB runs on must support crash consistent snapshot feature.

Reply:
Thanks @xumengpanda for the comments. I was still doing some final touches, but good to get some early comments.

Reply:
tweaked it to have support for disk snapshots too.